app/src/main/java/learning/android/tenmarks/com/androidlearning/SettingsActivity.java

```diff
@@ -1,10 +1,12 @@
 package learning.android.tenmarks.com.androidlearning;
 
 import android.app.Activity;
+import android.content.Intent;
 import android.media.audiofx.BassBoost;
```

Comment:
shortcut key OPTION + CONTROL + O for ordering imports and removing unused imports.